cmd/tester/main.go

```diff
@@ -37,6 +49,7 @@ func main() {
 	}
 
 	err := t.Test(invoker)
+	events.Log("Test result file: %v", config.TestResultFile)
```

Comment:
Is this from debugging?

Reply:
No, it's deliberate. The idea is to announce where the results are written to when testing finishes, if users want to view it.